libraries/aws_db_subnet_group.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsDbSubnetGroup < AwsResourceBase
+  name 'aws_db_subnet_group'
+  desc 'Verifies settings for a Subnet Group'
+  example "
+    describe aws_subnet_group(db_subnet_group_name: 'subnet-group') do
+      it { should exist }
+    end
+  "
+
+  def initialize(opts = {})
+    opts = { db_subnet_group_name: opts } if opts.is_a?(String)
+    super(opts)
+    validate_parameters(required: [:db_subnet_group_name])
+    @display_name = opts[:db_subnet_group_name]
+
+    catch_aws_errors do
+      resp = @aws.rds_client.describe_db_subnet_groups({ db_subnet_group_name: opts[:db_subnet_group_name] })
+      @subnet_group = resp.db_subnet_groups[0].to_h
+
```

Comment:
It might be nice to use the newly added `empty_response_warn`. This was added in @omerdemirok recent PR...
https://github.com/inspec/inspec-aws/pull/213/files#diff-39516680ba5f1844495fd773511eefc6R42

Reply:
Updated in recent commit